### PR TITLE
passphrase + README

### PR DESCRIPTION
**Pull request by if0s**
https://github.com/elasticio/sftp-component/issues/117
* Add support `passphrase` to credentials
* README improvements

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,9 +21,9 @@
     "async": "3.2.3",
     "elasticio-node": "0.0.9",
     "elasticio-rest-node": "1.2.7",
-    "elasticio-sailor-nodejs": "2.6.27",
+    "elasticio-sailor-nodejs": "2.6.28",
     "micromatch": "4.0.2",
-    "moment": "2.29.2",
+    "moment": "^2.29.4",
```
Comment (EmptyInfinity) [inline package.json:26]:
```suggestion
    "moment": "2.29.4",
```

File: component.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "title": "SFTP",
   "buildType": "docker",
   "description": "Provides file access and transfer using SSH File Transfer Protocol",
-  "version": "1.4.9",
+  "version": "1.5.0-dev.2",
```
Comment (EmptyInfinity) [inline component.json:5]:
```suggestion
  "version": "1.4.10-dev.2",
```

Comment (if0s) [reply component.json:5]: New functionality and changes in UI (added passphrase field)

File: component.json [hunk 3]
```diff
@@ -36,6 +36,13 @@
         "required": false,
         "placeholder": "Paste your SFTP Private Key",
         "note": "For using Private Key based authentication, leave Password field empty"
+      },
+      "passphrase": {
+        "viewClass": "PasswordFieldView",
+        "label": "Passphrase",
+        "required": false,
+        "placeholder": "Paste your passphrase",
+        "note": "If Private Key use passphrase, put it here"
```
Comment (A3a3e1) [inline component.json:45]:
```suggestion
        "note": "If private key is protected by a passphrase, put it here"
```

File: component.json (hunk 2 above)
Comment (EmptyInfinity) [reply component.json:5]: I see. my bad

**Review by EmptyInfinity — APPROVED** [review]

**Pull request merged**